README.md

```diff
@@ -1,13 +1,53 @@
-# saspy
-Slots accounting system protocol based on Python for arm architecture
+# SASPY
+Slots Accounting System (SAS) Protocol is a standard when comes to Slot Machine and VLTs.
 
-The SAS library was made using python and can be used for linux-based single-board computers (Raspberry, Orange PI, Banana Pi).
-The Library is used for connecting to EGM by RS-232 protocol and for receiving and processing some data of SAS-protocol.
+## How To
+```bash
+$ git clone https://github.com/zacharytomlinson/saspy.git
+$ cd saspy
+$ pip install -r requirements.txt
+$ nano config.yml # Adjust according your needs
+$ python example.py
+```
 
-SAS  (Slot  Accounting  System)  protocol  is  the  de  facto  casino communications  standards  designed  to  automate  slot  machine  meter  reporting  and event  logging,  player  tracking,  bonusing,  ticketing  and  cashless  gaming. 
+# Notes
+## Raspberry Hardware Limitation
+As per SAS documentation, for the connection, you need:
+> 19.2 KBaud in a "wakeup" mode. The 11-bit data packet consists of one start bit, eight data bits, a
+> ninth ‘wakeup’ bit, and one stop bit.
 
-The EGM and SMIB communicate each other in SAS protocol standards, whereas  communication  between  the  SMIB  and  host  is  not  standardized,  which interoperability  and  standards  issue  is  resolved  by  new  standards,  G2S  (Game to System)  protocol.  SMIB also  provides touch  screen and  card readers  for AFT  and player tracking service. 
+I won't go into detail on why the raspberry will never work with these settings BUT i'm gonna share a link that you can read whenever you want that explains WHY. [HERE](https://raspberrypi.stackexchange.com/questions/45570/how-do-i-make-serial-work-on-the-raspberry-pi3-pizerow-pi4-or-later-models/107780#107780) for the explanation.
```

Comment:
I have it working on a raspberry pi 3 and given the same settings I believe I can run it on a 4 as well. I do not have a 4 next to me at the moment but I will test when home. Just a clarification we can probably advise how to configure the pi to use full UART so long as bluetooth and console alternatives to tty1 are disabled.